en/simulation/simulation-in-hardware.md

```diff
@@ -14,34 +14,45 @@ The SIH provides two benefits over the HITL:
   They can, for instance, modify the aerodynamic model, or noise level of the sensors, or even add a sensor to be simulated.
 
-The SIH can be used by new PX4 users to get familiar with PX4 and the different modes and features, and of course to learn to fly a quadrotor with the real RC controller.
+The SIH can be used by new PX4 users to get familiar with PX4 and the different modes and features, and of course to learn to fly a quadrotor or an airplane with the real RC controller.
 
-The dynamic model is described in this [pdf report](https://github.com/PX4/Devguide/raw/master/assets/simulation/SIH_dynamic_model.pdf).
+The quadrotor dynamic model is described in this [pdf report](https://github.com/PX4/PX4-user_guide/raw/master/assets/simulation/SIH_dynamic_model.pdf).
+
+The aerodynamic model for the fixed-wing airplane is inspired from the PhD thesis:  
+> "Dynamics modeling of agile fixed-wing unmanned aerial vehicles." Khan, Waqas, supervised by Meyer Nahon, McGill University, PhD thesis, 2016.
 
 Furthermore, the physical parameters representing the vehicle (such as mass, inertia, and maximum thrust force) can easily be modified from the [SIH parameters](../advanced_config/parameter_reference.md#simulation-in-hardware).
 
 ## Requirements
 
-To run the SIH, you will need a [flight controller hardware](../flight_controller/README.md) (e.g. a Pixhawk-series board).
-If you are planning to use a [radio control transmitter and receiver pair](../getting_started/rc_transmitter_receiver.md) you should have that too.
-Alternatively, using *QGroundControl*, a [joystick](https://docs.qgroundcontrol.com/en/SetupView/Joystick.html) can be used to emulate a radio control system.
+To run the SIH, you will need a [flight controller](../flight_controller/README.md) (e.g. a Pixhawk-series board).
+you will also need either a [radio control transmitter and receiver pair](../getting_started/rc_transmitter_receiver.md) or a [joystick](https://docs.qgroundcontrol.com/en/SetupView/Joystick.html) (a joystick can be used via QGroundControl to emulate a radio control system).
 
-The SIH is compatible with all Pixhawk-series boards except those based on FMUv2.
-It is available on the PX4-Autopilot master branch and release versions v1.9.0 and above.
+- SIH is compatible with all Pixhawk-series boards except those based on FMUv2.
+- SIH for quadrotor supported from PX4 v1.9.
+- SIH for fixed-wing (airplane) is supported in versions after PX v1.12 (currently in the master branch).
 
 ## Setting up SIH
 
 Running the SIH is as easy as selecting an airframe.
-Plug the autopilot to the desktop computer with a USB cable, let it boot, then using a ground control station select the [SIH airframe](../airframes/airframe_reference.md#simulation-copter).
+Plug the autopilot to the desktop computer with a USB cable, let it boot, then using a ground control station select the [SIH Quadcopter X](../airframes/airframe_reference.md#simulation-copter) or the `SIH plane AERT`.
```

Comment:
`SIH plane AERT` and be linked once the changes go into firmware (this will push the reference for the airframe into docs)

Reply:
I'm going to merge. Can't fix this yet as airframes have not yet updated: https://github.com/PX4/PX4-Autopilot/pull/17991#issuecomment-896458873